spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -3,20 +3,34 @@
 RSpec.feature "Brexit Checker workflow", type: :feature do
   include BrexitCheckerHelper
 
-  scenario "business questions" do
+  scenario "business and citizen questions" do
     when_i_visit_the_brexit_checker_flow
     and_i_answer_citizen_questions
     and_i_answer_business_questions
     then_i_should_see_the_results_page
+    and_i_should_see_the_citizens_action_header
```

Comment:
To avoid adding more methods in here, could we look at combining them into a single 'then' step where they 'see_citizen_and_business_results', etc?

Reply:
How do these rspec feature tests work, are they just looking for each line to be `true`?

So you'd use `!see_citizen_and_business_results` and `see_citizen_and_business_results` in the main method?